src/Microsoft.Health.Fhir.Api/Features/Throttling/ThrottlingMiddleware.cs

```diff
@@ -274,7 +275,7 @@ private async Task Return429(HttpContext context)
         {
             Interlocked.Increment(ref _currentPeriodRejectedCount);
 
-            _logger.LogWarning($"{Resources.TooManyConcurrentRequests}. Limit is {_concurrentRequestLimit}.");
+            _logger.LogWarning($"{Resources.TooManyConcurrentRequests} Limit is {_concurrentRequestLimit}. Requests in flight {_requestsInFlight}");
```

Comment:
Worth noting also that when using string interpolation, it will prevent the variables showing up in structured logging

Reply:
+1 String parsing the log message is less than ideal.